projects/js-packages/ai-client/src/hooks/use-ai-suggestions/index.ts

```diff
@@ -256,9 +256,18 @@ export default function useAiSuggestions( {
 		( event: CustomEvent ) => {
 			closeEventSource();
 
-			const fullSuggestion = removeLlamaArtifact( event?.detail );
+			let message = event?.detail;
+			let skipRequestCount = false;
+			if ( event?.detail?.message ) {
+				message = event?.detail?.message;
```

Comment:
Was this broken before or is it a change introduced with the ChromeAI implementation?
If it is necessary, it can also be simplified: `let message = event?.detail?.message ?? event?.detail;`

Reply:
It's a change I added for this PR because I had to change what was in the "done" custom event's `detail` prop in order to indicate the event came from Chrome AI, but also wanted to maintain backwards compatibility. We can change the structure for done events entirely though, since I agree this is kind of weird.

Reply:
Ah, yes, I quickly read it and thought it was an object with only one key before. Ignore that :man_facepalming: 

That said, the simplification above should remove any ambiguity, no need to change the structure IMO